### DISCUSSION TITLE
Should I need to delete the plots which quality <0.7

### DISCUSSION THREAD

**Discussion by anderzheng** [Chia-Network/chia-blockchain#4030] [General]
I check my plots whith chia plots check, and some plots's quality is less than 0.7, even 0.3.
Should I keep them if my disk space is enough?
or will the low quality plots reduce the chance to win?

**Comment by rodollfomartins** [comment]
how to verify ?

**Reply by anderzheng** [reply]
use cli command:
chia plots check

**Comment by Team4N6** [comment]
Having extra plots, even low quality ones, increases your chance to win. However, at some point when you don't have the disk space, pruning low quality plots may be worthwhile.

I'm still not sure how closely the tests relate to real world results though.

**Reply by anderzheng** [reply]
I think so, too. But not sure.

**Comment by rodollfomartins** [comment]
my minimun is 0.80 and maximun 1.22

**Reply by rodollfomartins** [reply]
with -n100

**Reply by anderzheng** [reply]
from the wiki, if the quality > 0.7， it's good plots.

**Comment by chiabug** [comment]
I think the quality number is just a function of the challenge hashes used to check the plots during that invocation of the command.  Using "check" a day or two later will generate a different quality rating.  The actual rating should not be relevant to the real world as long as the actual challenge hashes sent by the network are truly random.
